requirements.txt

```diff
@@ -0,0 +1,6 @@
+black==25.1.0
```

Comment:
only keep this req please

note, keep this only in the requirements.txt for developers. it is not needed for building the package in pyproject.toml